src/Auth.spec.ts

```diff
@@ -1408,7 +1410,7 @@ describe('Auth', () => {
     auth.service.userName = 'user';
     auth.service.password = 'pwd';
     auth.service.certificate = 'cert';
-    auth.service.thumbprint = 'thumb';
+    auth.service.certificate = 'thumb';
```

Comment:
Is this intentional? If so, why set the same value above to `cert`?

Reply:
No, this looks like something that went wrong when reverting my "removal of thumbprint" action. I reverted some of my files (partially), my suspicion is that git and I just botched this line 😃